### DISCUSSION TITLE
Warning for Plex script: "You do not have permission to access this server"

### DISCUSSION THREAD

**Discussion by sarpuser** [tteck/Proxmox#3503] [General]
I just spent HOURS trying to get a brand new Plex server working. When I went to `http://\<plex-ip\>:32400/web' I kept getting an error saying "You do not have permission to access this server". Upon some research, it seemed that everyone was having this issue with older servers, and there was a lot of misinformation out there. I FINALLY figured out that **Plex restricts initial setup to localhost <ins>ONLY.</ins>** For some reason, the geniuses at Plex have blocked initial setup from an IP other than localhost :/

If you come across this issue (you will if you are doing a new install), here is how to fix it:
1. On the Plex LXC, enable root ssh. To do this you need to edit `/etc/ssh/sshd_config` and uncomment (remove the "#") the line that says `#PermitRootLogin prohibit-password`, and change `prohibit-password` to `yes`
2. On a device that can open a web browser, go to the terminal and start an ssh tunnel to the plex server by typing `ssh -L 32400:localhost:32400 root@<plex-server-IP>`
3. Go to a browser and type in `localhost:32400/web`
4. Log into your account and finish setup
5. Go back to your terminal where you started the SSH tunnel and type `exit` to close the session and log out

Since this seems to be problem on Plex's side with new servers, I would like to ask the development team of the helper scripts to please add some sort of warning in the script description, in case people come across this in the future. This is only an issue when building a brand new server, and does not seem to happen when migrating (auth tokens are also copied).

**Comment by taken2jail** [comment]
on the LXC container, it ask for password and login. I cant even log in to type in the commands for ssh. ask ideas?

**Reply by sarpuser** [reply]
> on the LXC container, it ask for password and login. I cant even log in to type in the commands for ssh. ask ideas?

log into the proxmox server with the root@pam account and type "passwd" to change your root password

**Reply by taken2jail** [reply]
hi thanks for the quick reply. I am login into the proxmox server.  However in plex counsel, it ask for login and password. i tried to use my proxmox login and password but it didn't work. Do you mean using the command in Shelll? because that didnt work either

**Reply by tteck** [reply]
❗ It is crucial not to exit the Proxmox shell while running scripts. Leaving the shell during script execution will result in the script being interrupted.

**Reply by tteck** [reply]
Sounds like the script did not complete successfully

**Reply by sarpuser** [reply]
> Sounds like the script did not complete successfully 

is that regarding my post or the other person's reply

**Reply by tteck** [reply]
@taken2jail

**Reply by taken2jail** [reply]
@tteck @sarpuser you're correct that the script didnt finish so I tried again and able to get onto the web:32400 with the "you do not have permission." I went into the LXC to type in the first command and i get the message "permission denied." any idea how to set permission to modify the file?

**Reply by sarpuser** [reply]
you need to be logged in as root in the lxc container. If you log into proxmox using the root@pam account you will be automatically logged into the lxc console as root as well. Then you can execute passwd to change the root password, and edit the sshd_config file. Then you should be set to go fo setting up the SSH tunnel and claiming your server.

**Reply by taken2jail** [reply]
Hi, thanks for the info. Unfortunately, the "/etc/ssh/sshd_config" gave me a permission denied message. I was able to work around it by using the ADVANCE mode during installation to set the SSH password.

 Once I was in, I went straight to step 2,3, and 4 to claim the web browser. I stumped into another issue after trying to set up the rest of plex. Plex asked for a monthly/annual/ or life time membership and I wasn't able to x out of the subscription.

**Reply by acc3366** [reply]
@sarpuser 
Many thanks to you for the solution. I have spent many sleepless nights trying to fix this issue.
